odoo/tools/date_utils.py

```diff
@@ -252,3 +252,4 @@ def date_range(start, end, step=relativedelta(months=1)):
     while dt <= end:
         yield localize(dt)
         dt = dt + step
+
```

Comment:
:thinking:

Reply:
It was a mistake. I have no intend to modify this file.